### DISCUSSION TITLE
Recovery from a DVR system

### DISCUSSION THREAD

**Discussion by fewirchen** [sepinf-inc/IPED#350] [Q&A]
Please,

In a scenario in which the HD was removed from a common DVR system and there is a need to recover a video file (a footage) that has just been overlaid (only once, very recently) by rotation due to the lack of it space, would I be successful with the execution of IPED? If so, what would be the syntax and / or profile to be used?

**Comment by lfcnassif** [comment]
Currently DVR file systems are not supported by Sleuthkit, so IPED doesn't support them too. I suggest you to look for a DVR recovery specific tool, but if the video was overwritten, there is little chance of a successful recovery.
